templates/helpers.rb

```diff
@@ -118,6 +118,21 @@ def section_title(sec = self)
     end
   end
 
+  def revealjs_dependencies(document, node, revealjsdir)
+    dependencies = []
+    dependencies << "{ src: '#{revealjsdir}/plugin/highlight/highlight.js', async: true }" if (document.attr? 'source-highlighter', 'highlightjs')
```

Comment:
Note to self: Now that #320 is in. This needs to be removed (Highlight.js is injected via docinfo) in addition to handle the conflict.